setup.py

```diff
@@ -0,0 +1,238 @@
+    "pymedphys.tests.experimental.pinnacle",
+    "pymedphys.tests.experimental.profiler",
+    "pymedphys.tests.experimental.pseudonymisation",
+    "pymedphys.tests.gamma",
+    "pymedphys.tests.logfiles",
+    "pymedphys.tests.logging",
+    "pymedphys.tests.losslessjpeg",
+    "pymedphys.tests.mocks",
+    "pymedphys.tests.monaco",
+    "pymedphys.tests.mudensity",
+    "pymedphys.tests.trf",
+    "pymedphys.tests.winstonlutz",
+]
+
+package_data = {
+    "": ["*"],
+    "pymedphys._experimental.serviceplans": ["templates/*"],
+    "pymedphys.tests.dicom": ["data/rtplan/*", "scratch/*"],
+    "pymedphys.tests.e2e": [
+        "cypress/*",
+        "cypress/integration/streamlit/*",
+        "cypress/plugins/*",
+        "cypress/support/*",
+    ],
+    "pymedphys.tests.experimental.mephysto": [
+        "data/baselines/*",
+        "data/measurements/*",
+    ],
+    "pymedphys.tests.experimental.paulking.film": ["data/*"],
+}
+
+extras_require = {
+    ':python_version >= "3.6.0" and python_version < "3.7.0"': ["dataclasses"],
+    "dev": [
+        "tqdm",
+        "attrs",
+        "watchdog",
+        "keyring",
+        "packaging",
+        "PyYAML",
+        "requests",
+        "numpy>=1.12",
+        "matplotlib",
+        "scipy",
+        "pandas",
+        "Pillow",
+        "imageio",
+        "shapely>=1.7.0",
+        "dbfread",
+        "pydicom>=2.0.0",
+        "pynetdicom",
+        "pylibjpeg-libjpeg",
+        "immutables",
+        "cython",
+        "pymssql",
+        "python_dateutil",
+        "scikit-image",
+        "streamlit==0.70.0",
+        "timeago",
+        "sphinx>=1.4,<1.8",
+        "sphinx-rtd-theme>=0.4.3,<0.5.0",
+        "sphinxcontrib-napoleon",
+        "sphinx-argparse",
+        "sphinx-autobuild",
+        "nbsphinx",
+        "jupyter_client",
+        "ipython",
+        "ipykernel",
+        "nbconvert==6.0.6",
+        "pytest",
+        "pytest-sugar",
+        "hypothesis",
+        "psutil",
+        "pylint",
+        "pytest-pylint",
+        "pre-commit",
+        "black>=19.3b0,<20.0",
+        "mypy",
+        "rope",
+        "doc8",
+        "tomlkit",
+    ],
+    "dicom": ["pydicom>=2.0.0", "pynetdicom", "pylibjpeg-libjpeg", "immutables"],
+    "docs": [
+        "sphinx>=1.4,<1.8",
+        "sphinx-rtd-theme>=0.4.3,<0.5.0",
+        "sphinxcontrib-napoleon",
+        "sphinx-argparse",
+        "sphinx-autobuild",
+        "nbsphinx",
+        "jupyter_client",
+        "ipython",
+        "ipykernel",
+        "nbconvert==6.0.6",
+    ],
+    "doctests": ["tensorflow>=2.2.0", "black>=19.3b0,<20.0", "tomlkit"],
+    "experimental": ["python_dateutil", "scikit-image"],
+    "gui": ["streamlit==0.70.0", "timeago"],
+    "ml": ["tensorflow>=2.2.0"],
+    "tests": [
+        "pytest",
+        "pytest-sugar",
+        "hypothesis",
+        "psutil",
+        "pylint",
+        "pytest-pylint",
+    ],
+    "user": [
+        "tqdm",
```

Comment:
As an fyi, I find I have to do
sudo apt-get install tqdm
otherwise running the tests fail for the lack of tqdm

Reply:
Cheers for the heads up. 

I suspect that may be due to Python path issues. I actually make sure that absolutely nothing is installed within the root Python to make it clear when my python path is being mangled.